backend/app/assets/stylesheets/spree/backend/globals/_variables.scss

```diff
@@ -10,59 +10,68 @@ $font-size-pill: 11px;
 //--------------------------------------------------------------
 
 // Basic color palette for admin
-$color-1:                        #FFFFFF !default;    // White
-$color-2:                        #9FC820 !default;    // Green
-$color-3:                        #5498DA !default;    // Light Blue
-$color-4:                        #6788A2 !default;    // Dark Blue
-$color-5:                        #C60F13 !default;    // Red
-$color-6:                        #FF9300 !default;    // Yellow
-$color-7:                        #F4F4F4 !default;    // Light Gray
```

Comment:
I've just one concern about removing variables. If extensions or store were using them they'll end up having an error while compiling. What about moving them into a `_deprecated-variables.scss` file instead? 

I'm not sure if there's a way to actually deprecate them with scss. I've made some test with this code:

```scss
@if variable-exists(color-1) {
  @warn '$color-1 will be removed, please use $color-primary;';
  $color-1: $color-primary;
}
```

and it works, but it only prints a deprecation message if you re-define `$color-1`, not if you just use it. It also has to be declared **after** the `$color-1` definition, which is basically useless for both extensions and stores since they will eventually declare/override variables after solidus files are loaded.

@tvdeyen do you have any idea around this?

Reply:
Great point @kennyadsl 

We already have a way to deprecate variables https://github.com/solidusio/solidus/blob/master/backend/app/assets/stylesheets/spree/backend/globals/_deprecation.scss

You use it like this https://github.com/solidusio/solidus/blob/475d9db5d0291dd4aeddc58ec919988c336729bb/backend/app/assets/stylesheets/spree/backend/globals/_variables.scss#L42

Reply:
Hi guys! I've just pushed a commit to resolve this issue; as you can seen the old numbered color variables has been re-added in a their own scss partial, and I used the existing sass mixin, suggest by @tvdeyen (I just duplicate it and renamed the old one because it refers to Bootstrap variables, and its warning message, also). 
Thank you @kennyadsl for your feedback!

The message that I wrote refers to a future 3.0 version of Solidus, so we can keep the deprecated variables on 2.8 and 2.9vs. What do you think about that?

Reply:
Awesome 🍰